hooks/993-factory-files.chroot

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+# tmpfiles.d will copy files from /usr/share/factory So file that
+# needs to be populated in /writable have to be copied there.
```

Comment:
I think it would be good to add a more extensive comment mentioning `core-writable.conf` and a quick mention of what "C" and "d" options do, and `s/tmpfiles.d/tmpfiles.d(5)/` so it is clear that the man page can be consulted for details.